python/feathub/feature_views/transforms/python_udf_transform.py

```diff
@@ -24,14 +24,48 @@ class PythonUdfTransform(Transformation):
     time.
     """
 
-    def __init__(self, udf: Callable[[pd.Series], Any]) -> None:
+    def __init__(
+        self,
+        udf: Callable[[pd.Series], Any],
+        fail_on_exception: bool = True,
```

Comment:
A similar configuration I've seen is `HasHandleInvalid` in the Flink ML project, which has the following strategies when dealing with invalid values or errors:
- `error`: throws out the internal exception
- `keep`: outputs a default value
- `skip`: filters out such rows and do not output them

Do you think we should add an option that corresponds to the `skip` strategy in Feathub?

Reply:
I think `skip` strategy may not be a good strategy to use in Feathub. In Feathub, user defines the computation logic of each feature, and each feature is a column in the row. I think skipping the entire row when one of the features is invalid is undesirable.  

I think it makes sense in FlinkML, because in FlinkML the input is a row and the output is also a row.